tests/views/test_collection_exercise.py

```diff
@@ -794,13 +798,55 @@ def test_remove_loaded_sample_failed(self, mock_request):
         )
 
         self.assertEquals(response.status_code, 200)
-        self.assertIn("Error failed to remove sample".encode(), response.data)
+        self.assertIn('Error: Failed to remove sample'.encode(), response.data)
 
-    @requests_mock.mock()
-    def test_get_confirm_remove_sample(self, mock_request):
-        response = self.app.get(
-            f"/surveys/{short_name}/{period}/confirm-remove-sample", follow_redirects=True
-        )
+    def test_get_confirm_remove_sample(self):
+        response = self.app.get(f"/surveys/test/000000/confirm-remove-sample",
+                                follow_redirects=True)
+
+        self.assertEquals(response.status_code, 200)
+        self.assertIn("Remove sample from test 000000".encode(), response.data)
+
+    @mock.patch("response_operations_ui.controllers.survey_controllers.get_survey", return_value=survey_by_id)
+    def test_get_create_ce_event_form_success(self, survey):
+
+        response = self.app.get(f"/surveys/MBS/201901/{collection_exercise_id}/confirm-create-event/mps",
+                                follow_redirects=True)
+
+        self.assertEquals(response.status_code, 200)
+        self.assertIn("MBS".encode(), response.data)
+        self.assertIn("Main print selection".encode(), response.data)
+
+    @mock.patch("response_operations_ui.controllers.collection_exercise_controllers.get_collection_exercise")
+    @mock.patch("response_operations_ui.controllers.collection_exercise_controllers.create_collection_exercise_event")
+    def test_create_collection_exercise_event_success(self, mock_create_ce_event, mock_get_ce_details):
+        mock_get_ce_details.return_value = ce_details_no_events
+        create_ce_event_form = {
+            "day": "01",
+            "month": "01",
+            "year": "2018",
+            "hour": "01",
+            "minute": "00"
+        }
+
+        response = self.app.post(f"/surveys/MBS/201901/{collection_exercise_id}/create-event/mps",
+                                 data=create_ce_event_form, follow_redirects=True)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertIn("Event date added.".encode(), response.data)
+
+    @mock.patch("response_operations_ui.controllers.survey_controllers.get_survey", return_value=survey_by_id)
```

Comment:
For consistency you should make this `@requests_mock.mock.patch`

Reply:
@MIslam92 different type of patch. 

`@requests_mock.mock.patch` is for bringing in a URL patcher.

The standard `@unittest.mock.patch` is for everything else.

Reply:
ahh sorry my bad got mixed up thanks for clearing up